opensrp-anc/src/main/java/org/smartregister/anc/library/util/ANCFormUtils.java

```diff
@@ -543,7 +544,7 @@ static String cleanValue(String value) {
                         JSONObject jsonObject = jsonArray.optJSONObject(i);
                         if (jsonObject != null && !jsonObject.optString(JsonFormConstants.TEXT).isEmpty()) {
                             String text = jsonObject.optString(JsonFormConstants.TEXT), translatedText = "";
-                            translatedText = NativeFormLangUtils.getTranslatedString(text);
+                            translatedText = NativeFormLangUtils.getTranslatedANCString(text, AncLibrary.getInstance().getApplicationContext());
```

Comment:
Please extend the NF NativeFormLangUtils class if this functionality can only be used by ANC.  I think this can be used in other scenarios by other apps. 

Please rename the method to something ling `getSpecificTranslationString` Also write JavaDocs on what it achieves